etc/kayobe/pulp-repo-versions.yml

```diff
@@ -18,6 +18,26 @@ stackhpc_pulp_repo_rhel_9_mariadb_10_11_version: 20250205T001351
 stackhpc_pulp_repo_rhel_9_rabbitmq_erlang_version: 20240711T091318
 stackhpc_pulp_repo_rhel_9_rabbitmq_server_version: 20240711T091318
 stackhpc_pulp_repo_rhel_9_treasuredata_5_version: 20241115T002028
+stackhpc_pulp_repo_rocky_9_1_appstream_version: 20231207T013715
+stackhpc_pulp_repo_rocky_9_1_baseos_version: 20231206T014015
+stackhpc_pulp_repo_rocky_9_1_crb_version: 20231211T120328
+stackhpc_pulp_repo_rocky_9_1_extras_version: 20231211T120328
+stackhpc_pulp_repo_rocky_9_1_highavailability_version: 20231208T011702
+stackhpc_pulp_repo_rocky_9_2_appstream_version: 20231211T120328
+stackhpc_pulp_repo_rocky_9_2_baseos_version: 20231211T120328
+stackhpc_pulp_repo_rocky_9_2_crb_version: 20231207T013715
+stackhpc_pulp_repo_rocky_9_2_extras_version: 20231207T013715
+stackhpc_pulp_repo_rocky_9_2_highavailability_version: 20231208T011702
+stackhpc_pulp_repo_rocky_9_3_appstream_version: 20240508T001430
+stackhpc_pulp_repo_rocky_9_3_baseos_version: 20240509T005935
+stackhpc_pulp_repo_rocky_9_3_crb_version: 20240429T003818
+stackhpc_pulp_repo_rocky_9_3_extras_version: 20240506T000343
+stackhpc_pulp_repo_rocky_9_3_highavailability_version: 20240510T001129
+stackhpc_pulp_repo_rocky_9_4_appstream_version: 20240816T002610
+stackhpc_pulp_repo_rocky_9_4_baseos_version: 20240816T002610
+stackhpc_pulp_repo_rocky_9_4_crb_version: 20240816T002610
+stackhpc_pulp_repo_rocky_9_4_extras_version: 20240816T002610
+stackhpc_pulp_repo_rocky_9_4_highavailability_version: 20240816T002610
```

Comment:
I think we can get rid of anything older than RL 9.5 here

Reply:
We decided to keep supporting older versions as customers probably need 9.4 or older to avoid possible issue with OFED.